addons/test_website/static/tests/tours/website_page_properties.js

```diff
@@ -107,27 +123,30 @@ function checkIsTemplate(isTemplate, pageTitle = undefined) {
     ];
 }
 
+const testEditMenuDialog = [
```

Comment:
Since removing the previous commit, this is conflicting. I'll remove the testing part of the commit for now.